crates/proto/src/op/message.rs

```diff
@@ -822,10 +794,8 @@ pub struct MessageParts {
     pub name_servers: Vec<Record>,
     /// message additional records
     pub additionals: Vec<Record>,
-    /// sig0 or tsig
-    // this can now contains TSIG too. It should probably be renamed to reflect that, but it's a
-    // breaking change
-    pub sig0: Vec<Record>,
+    /// message signature
+    pub signature: MessageSignature,
```

Comment:
Mind extracting the renaming of fields and records into a separate commit, so we keep the more interesting type introduction separate from the mechanical renaming?

Reply:
Done.